rtrlib/aspa/aspa.c

```diff
@@ -234,29 +254,49 @@ static enum aspa_status aspa_table_update_internal(struct aspa_table *aspa_table
 	size_t unused_capacity = 0;
 	size_t existing_i = 0;
 
+#ifndef ASPA_UPDATE_IN_PLACE
+	uint32_t last_asn = 0;
+	bool current_withdrawn = false;
+#endif
```

Comment:
this seems redundant...

Reply:
and `last_asn` shouldn't be initialized with zero.... either store the last record (which I'd try to a void in the first place) and then check for `NULL`.

Reply:
`current_withdrawn` can easily be checked by comparing the flag in the current op... and even then, there're enum cases for that kind of differentiation.

Reply:
In the update function I'm in favor of either referring to `add` and `remove` operations or `announce` and `withdraw` operations. Either way, we should remain consistent with the terminology here.